app/models/invoice.rb

```diff
@@ -11,6 +11,7 @@ def customer_name
   end
 
   def merchant_items(merchant)
+    # require "pry"; binding.pry
```

Comment:
let's delete this before we forget about it!

Reply:
Removed!